src/heretic/mn_menu.c

```diff
@@ -483,6 +512,14 @@ static const multiitem_t multiitem_difficulties[NUM_SKILLS] =
     {SKILL_HNTR, "YELLOWBELLIES"},
 };
 
+multiitem_t multiitem_translucency[NUM_TRANSLUCENCY] =
+{
+    {TRANSLUCENCY_OFF, "OFF"},
+    {TRANSLUCENCY_MISSILE, "PROJECTILES"},
+    {TRANSLUCENCY_ITEM, "VANILLA WEAPON FLASH"},
```

Comment:
I used the item for the vanilla weapon flash instead, to have the "both" option (binary 11) working correctly.

Reply:
Fine with me, but please drop the "Vanilla" part. This is an implementation detail that needn't get exposed in the in-game menu.